### DISCUSSION TITLE
44100Hz Ja Support

### DISCUSSION THREAD

**Discussion by tonnetonne814** [p0p4k/vits2_pytorch#20] [Show and tell]
This is a great implementation.

Based on this repository, modified codes so that 44100Hz Japanese speech can be learned, 
and learning is progressing without any problems.
A pretrain model will also be added, later.

https://github.com/tonnetonne814/unofficial-vits2-44100-Ja

I hope this helps.

**Comment by nikich340** [comment]
Actually no changes are required to repo to support any language in any sample rate. All you need is to prepare processed (phonemized, if needed) texts and corresponding audios in needed SR, then set "sampling_rate" param in config.
